yocto_dobi_gen.yaml

```diff
@@ -0,0 +1,69 @@
+#
+# This generates the yocto 
+#
+
+meta:
+  project: ci.os.lmp
+
+# ===================================================
+# mount points
+# ===================================================
+mount=source-dir:
+  bind: yocto
+  path: /data
+  read-only: false
```

Comment:
why needs to be read-write?

Reply:
Because the generated artifacts are also in yocto

Reply:
add comment why it needs r/w